### ISSUE TITLE
the output of bn3b1_branch2b is inf

### ISSUE THREAD

**Issue by jiaobingle** [antingshen/resnet-protofiles#16]
when i rewrite your code ResNet-101 for semantic segmentation (just changed the last several layers ),and train from scratch  or fine-tune with Kaiming He's caffemodel, it seems like this:
[Forward] Layer bn3a_branch2c, top blob res3a_branch2b data: inf

could you give me some suggestions, thx!

**Comment by antingshen** [comment]
probably the standard debugging steps for when a net runs into numerical limits, i.e. normalize data, lower learning rate, etc. It's hard to tell for your specific case.
